app/services/messages/application_submitted.rb

```diff
@@ -0,0 +1,35 @@
+module Messages
+  class ApplicationSubmitted
+    attr_reader :crime_application
+
+    def initialize(crime_application)
+      @crime_application = crime_application
+    end
+
+    def process
+      Rails.logger.info "==> Publishing application #{crime_application.id}"
+
+      sqs_client.send_message(
+        queue_url:,
+        message_body:,
+      )
+    end
+
+    private
+
+    # TODO: this is an early PoC, we can either instantiate
+    # a new client, or use the Shoryuken configured one,
+    # tho we incur in a bit of coupling.
+    def sqs_client
```

Comment:
Looks like the repo is looking for maintainers, the Gems last commit was in June last year and the latest version was in Feb 22.

Compared to a lot of gems this is till quite active so I'm not worried about the dependency per se - just something to bear in mind when we make the decision. 

Link https://github.com/ruby-shoryuken/shoryuken

Reply:
Yeah I was referring more to the fact I get the AWS SQS client through a bit of indirection from `Shoryuken` itself. When in reality we can built our own client with `Aws::SQS::Client.new(...)` 

But yep, I've researched quite a bit and this gem looks fantastic and is a direct drop-in replacement for Sidekiq (same interface, almost same configuration, etc.) and on top of that for jobs we adhere to ActiveJob to even less coupling with the underlying job processor implementation 👍